src/core/public/application/ui/app_not_found_screen.tsx

```diff
@@ -53,6 +57,8 @@ export const AppNotFound = () => (
                 id="core.application.appNotFound.pageDescription"
                 defaultMessage="No application was found at this URL. Try going back or choosing an app from the menu."
```

Comment:
If we add some text to have the user confirm that their default route advanced setting is valid we could also address https://github.com/opensearch-project/OpenSearch-Dashboards/issues/665.